packages/website/components/EcosystemSection.tsx

```diff
@@ -98,6 +98,14 @@ const ecosystemData: EcosystemData[] = [
       "ZKPool aggregates the computing power of accelerators for zero-knowledge proofs and provides services to ZKP applications.",
     filters: [],
   },
+  {
+    icon: "/images/ecosystem/kekkai.png",
+    name: "KEKKAI",
```

Comment:
we don't have an order for this now so we've just been doing lexicographic on name. can you please add in KEKKAI in lexicographic order and also perhaps a comment saying that?